app/components/gleam-logo.ts

```
@@ -0,0 +1,106 @@
+  };
+}
+
+export default class GleamLogoComponent extends Component<GleamLogoSignature> {
+  animationInterval: number | null = null;
+  container: HTMLElement | null = null;
+  @tracked riveInstance: Rive | null = null;
+
+  get containerStyle(): string {
+    // Ensure minimum size on mobile while maintaining aspect ratio
+    const minSize = Math.min(this.args.height, 200);
+    return `height: ${minSize}px; width: ${minSize}px; max-width: 100%;`;
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)